src/promptflow/promptflow/azure/_utils/gerneral.py

```diff
@@ -3,14 +3,69 @@
 # ---------------------------------------------------------
 
 import jwt
+from promptflow.exceptions import ValidationException
 
 
 def is_arm_id(obj) -> bool:
     return isinstance(obj, str) and obj.startswith("azureml://")
 
 
+def get_aml_token(credential):
```

Comment:
looks like we could call get_base_token in this function to reduce dulplicate code?

Reply:
done